TCS34725.h

```diff
@@ -77,7 +76,7 @@ class TCS34725_
     {
         wire = &w;
         uint8_t x = read8(Reg::ID);
-        if (x != ID_REG_PART_NUMBER) return false;
+        if (x != 0x44 && x != 0x4d) return false;
```

Comment:
I would appreciate your giving appropriate names and making them compile-time constants instead of the magic numbers.